File: IsItHalloween.java
import java.util.Scanner;
class IsItHalloween {
  public static void main(String[] args) {
    Scanner scan =new Scanner(System.in);
    String s=scan.nextLine();
    if(s.matches("OCT 31")||s.matches("DEC 25")){
    System.out.println("yup");
    }
    else{
          System.out.println("nope");

    }
  }
}

File: isithalloween.cpp
#include <iostream>

//#include <bits/stdc++.h>
using namespace std;

int main() {
  string m;
  int d;
  cin >> m >> d;
  if(m == "OCT" and d == 31 || m == "DEC" and d == 25){;
    cout << "yup";
  } else {
    cout << "nope";
  }
  return 0;
}